packages/graphql/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2.1'
+
+services:
+  mongo:
+    container_name: animavita-mongo
+    image: mongo
```

Comment:
I think it would be better to specify the version you are running it. just to avoid breaking changes